### ISSUE TITLE
Add support to the Elvis operator in S2930 (Dispose of unused fields)

### ISSUE THREAD

**Issue by njannink** [SonarSource/sonarlint-vs#202]
I get a false positive SonarLint issue on the following code:
```
namespace Test
{
    class MyClass : IDisposable
    {
        private Mutex _mutex; // false positive S2930

        public MyClass()
        {
             _mutex = new Mutex(true, "MyAppInstance");
        }

        public void Dispose()
        {
           _mutex?.Dispose();
           _mutex = null;
        }
    }
}
```

This is a false positive because SonarLint doesn't recognise the null operator (?). 

If I rewrite the code in the `Dispose` method to `if (_mutex != null) _mutex.Dispose();` Then the SonarLint issue is gone

**Comment by tvsonar** [comment]
@njannink Thanks for reporting this issue. We have scheduled it to the next sprint, we don't have a fixed timeline yet, but it should make into the master in the next 2-3 weeks.

**Comment by njannink** [comment]
You're welcome thanks for the feedback and telling me the (?) operator is actually called the elvis operator.
